composeApp/src/commonMain/kotlin/org/ooni/probe/data/models/BatteryState.kt

```diff
@@ -1,5 +1,7 @@
 package org.ooni.probe.data.models
 
-data class BatteryState(
-    val isCharging: Boolean,
-)
+enum class BatteryState {
+    NotCharging,
+    Charging,
```

Comment:
Maybe its better to check represent this as plugged into AC power. It happens that the device is plugged in but not charging.

Reply:
But the app could be running on a laptop that may or not be charging. And it's not easy to know that for us. That's why I want with Unknown.

Reply:
I see your point